core/src/test/scala/com/microsoft/azure/synapse/ml/Secrets.scala

```diff
@@ -59,14 +59,27 @@ object Secrets {
   lazy val MADTestConnectionString: String = getSecret("madtest-connection-string")
   lazy val MADTestStorageKey: String = getSecret("madtest-storage-key")
   lazy val MADTestSASToken: String = getSecret("madtest-sas-token")
-  lazy val SynapseExtensionPassword: String = getSecret("synapse-extension-dxt-password")
+
+  lazy val SecretRegexpFile: String = getSecret("secret-regexp-file")
+
   lazy val ArtifactStore: String = getSecret("synapse-artifact-store")
   lazy val Platform: String = getSecret("synapse-platform")
   lazy val AadResource: String = getSecret("synapse-internal-aad-resource")
-  lazy val SynapseExtensionTenantId: String = getSecret("synapse-extension-dxt-tenant-id")
-  lazy val SynapseExtensionUxHost: String = getSecret("synapse-extension-dxt-ux-host")
-  lazy val SynapseExtensionSspHost: String = getSecret("synapse-extension-dxt-ssp-host")
-  lazy val SynapseExtensionWorkspaceId: String = getSecret("synapse-extension-dxt-workspace-id")
+  lazy val SynapseExtensionEdogPassword: String = getSecret("synapse-extension-edog-password")
+  lazy val SynapseExtensionEdogTenantId: String = getSecret("synapse-extension-edog-tenant-id")
+  lazy val SynapseExtensionEdogUxHost: String = getSecret("synapse-extension-edog-ux-host")
+  lazy val SynapseExtensionEdogSspHost: String = getSecret("synapse-extension-edog-ssp-host")
+  lazy val SynapseExtensionEdogWorkspaceId: String = getSecret("synapse-extension-edog-workspace-id")
 
-  lazy val SecretRegexpFile: String = getSecret("secret-regexp-file")
+  lazy val SynapseExtensionDailyPassword: String = getSecret("synapse-extension-daily-password")
+  lazy val SynapseExtensionDailyTenantId: String = getSecret("synapse-extension-daily-tenant-id")
+  lazy val SynapseExtensionDailyUxHost: String = getSecret("synapse-extension-daily-ux-host")
+  lazy val SynapseExtensionDailySspHost: String = getSecret("synapse-extension-daily-ssp-host")
+  lazy val SynapseExtensionDailyWorkspaceId: String = getSecret("synapse-extension-daily-workspace-id")
+
+  lazy val SynapseExtensionDxtPassword: String = getSecret("synapse-extension-dxt-password")
+  lazy val SynapseExtensionDxtTenantId: String = getSecret("synapse-extension-dxt-tenant-id")
+  lazy val SynapseExtensionDxtUxHost: String = getSecret("synapse-extension-dxt-ux-host")
+  lazy val SynapseExtensionDxtSspHost: String = getSecret("synapse-extension-dxt-ssp-host")
+  lazy val SynapseExtensionDxtWorkspaceId: String = getSecret("synapse-extension-dxt-workspace-id")
```

Comment:
Theres a bit of duplication caused by this, perhaps we can make this more programmatic by specifying the deployment (dev, daily, weekly, prod) in a variable in this file and only getting secrets for that version this way we can reduce the repeated code and make the var names a bit shorter